src/Artsy/Router/RouterLink.tsx

```diff
@@ -9,15 +10,34 @@ import React from "react"
  *   ...
  * `
  */
-export const RouterLink: React.FC<LinkProps> = ({
-  to,
-  className,
-  children,
-  ...props
-}) => {
-  return (
-    <Link to={to} className={className} {...props}>
-      {children}
-    </Link>
-  )
+
+export const RouterLink: React.FC<LinkProps> = (
+  { to, className, children, ...props },
+  context
+) => {
+  // Check to see if we're within a router context. If not, fall back to a
+  // default link.
+  const isRouterContext = Boolean(context.router)
+
+  if (isRouterContext) {
+    return (
+      <Link to={to} className={className} {...props} data-router-to={to}>
+        {children}
+      </Link>
+    )
+  } else {
+    return (
+      <a href={to as string} className={className} {...props}>
+        {children}
+      </a>
+    )
+  }
+}
+
+/**
+ * FIXME: Upgrade to Found v.4: https://github.com/4Catalyzer/found/releases/tag/v0.4.0,
+ * which supports React 14.3 context style.
+ */
+RouterLink.contextTypes = {
+  router: PropTypes.any,
```

Comment:
Had to defer to old context behavior :/ Upgrading found will take some concerted attention.